pkgs/development/python-modules/watchfiles/default.nix

```
@@ -60,6 +60,10 @@ buildPythonPackage rec {
     cd tests
   '';
 
+  postUnpack = ''
```

```suggestion
  postPatch = ''
```

and move it up between cargoDeps and NativeBuildInputs so we we follow the chronological ordering of how the package is built.